shell/common/shorebird/shorebird.cc

```diff
@@ -80,6 +80,13 @@ FileCallbacks ShorebirdFileCallbacks() {
   };
 }
 
+void ConfigureShorebird(const ShorebirdFlutterProjectArgs& args,
```

Comment:
Neat!

Reply:
Thanks! It was Chinmay's suggestion that led to this :)